apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-checkbox.tsx

```diff
@@ -0,0 +1,47 @@
+import { FormFieldType, IFormField, IValueAndDisplay } from '@coscrad/api-interfaces';
+import { Checkbox } from '@mui/material';
+
+type CheckboxProps = IFormField & {
+    onIsCheckedChange?: (name: string, value: string | boolean) => void;
+    isChecked: boolean;
+};
+
+/**
+ * TODO[https://www.pivotaltracker.com/story/show/183941489]
+ * We may want to add an 'indeterminate' state so that the user can have
```

Comment:
If you're not doing this, you need to make sure the filter is actually defaulted to either `true` or `false.